packages/website/docs/MultilineBox.md

```diff
@@ -0,0 +1,45 @@
+# Best Practices
+The Multiline box allows users to **add the same sequence of inputs multiple times**.  
```

Comment:
@kpoulincoveo Before I review this, I would need to clarify the terminology we use. This is the component that allows users to enter multiple URLs when creating a web source, right?

```suggestion
The multiline box allows users to **provide several inputs of the same type**.  
```
or 
`The multiline box allows users to **provide a series of inputs of the same type**.`
Not sure what we mean with "sequence"... 🤔

Reply:
I Like the second option better personally :eyes:

Reply:
@jlevesquecoveo The multiline also serves to build complex conditions (ex: Filter expression) 

By "sequence", I mean that each line can hold multiple inputs (ex: first name + last name + email). That bundle of different input is a sequence. And each of line can be repeated multiple times (ex: to provide a list of people)

In some cases, the sequence of inputs can vary from one line to another (although it's not recommended to do it but some case allows it such as condition. That means that the first line could have a sequence of (field+comparator+single select) but the second line could have only (field+comparator+numerical input)


I let you decide which version of what you propose works best.

Reply:
OK, thanks for these details, @kpoulincoveo! Can the sequence contain only one input, though? Like this:
<img width="510" alt="image" src="https://user-images.githubusercontent.com/47796194/160683506-48711d4f-a2eb-42cf-a04f-4d58b4819e50.png">
Or is this a different component?

Reply:
Yes in that case it can (weither or not you use Multiline depend on the type of input, for instance, having a single select only on each line should be using a multi-select instead)